README.md

```diff
@@ -81,15 +81,15 @@ Under no circumstance does this plugin retain the private key for any certificat
 
 - The sub-path under `<mount>/issue-with-csr/` is intended for user-provided CSR
 
-This plugin will create a certificate within GCP CA Service with a certificate `Name` using the final path parameter in the Vault resource path.  For example, `gcppca/issue-with-genkey/my_tls_cert_rsa_1` will create a GCP CA Service Resource path `projects/your-project-id/locations/us-central1/certificateAuthorities/prod-root/certificates/my_tls_cert_rsa_1`.  This is the actual CA Service unique name for the certificate and cannot be reused once created.
+This plugin will create a certificate within GCP CA Service with a certificate `Name` using the final path parameter in the Vault resource path.  For example, `gcppca/issue-with-genkey/my_tls_cert_rsa_1` will create a GCP CA Service Resource path `projects/your-project-id/locations/us-central1/caPools/my-pool/certificates/my_tls_cert_rsa_1`.  This is the actual CA Service unique name for the certificate and cannot be reused once created.
 
 Deleting the key in Vault will revoke the certificate in CA Service which also means the same name cannot be reused.
 
 ### Vault Generated
 
 To generate a certificate keypair on vault, first apply a configuration that allows Vault to reference which CA to sign against 
 
-The configuration below will generate a certificate called `my_tls_cert_rsa_1` within CA Service using a GCP CA `prod-root` that was defined earlier by specifying `gcppca/config`.
+The configuration below will generate a certificate called `my_tls_cert_rsa_1` within CA Service using a GCP CA `prod-root` that was defined separately.  Note, `issuing_certificate_authority` is an optional parameter which if omitted, will cause the certificate to get issued by any CA in the pool.
```

Comment:
Can we remove the 'note'? As mentioned below, we'd like to encourage customers to not target specific CAs. I see that there is documentation for this parameter already.